### DISCUSSION TITLE
About Node.js version warning in 5.30.0 release notes

### DISCUSSION THREAD

**Discussion by justusbunsi** [verdaccio/verdaccio#4565] [Q&A] [answered]
Hi. Hopefully, this is the right place to ask this question. I read the [release notes for 5.30.0](https://github.com/verdaccio/verdaccio/releases/tag/v5.30.0) and am wondering, whether the Node.js warning applies to hosting Verdaccio or using it for pushing/pulling packages as a client?

If it applies to hosting Verdaccio, it shouldn't be an issue using the Docker image. Right?

> Be warned, if you are using Node.js 12 please do not upgrade to this version, the minimum is now Node.js 14 (due dependency reasons are out my hand), the recommendation is always go for [LTS releases](https://nodejs.org/en/about/previous-releases) anyway which at this moment is Node.js 20.

Thank you for clarifying.

**Comment by juanpicado** [comment, marked as answer]
@justusbunsi If your are using docker image no issue, inside docker file uses nodejs 20. 

It is only for npmjs users I will update the warning thanks :)

**Comment by juanpicado** [comment]
@KissMyPixel I appreciate the participation but every time you reply a lot of noise (part of the🧵 ) goes with your reply. Would you mind review your settings otherwise is really hard to interact with you.
